addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -12,7 +12,11 @@
                      data-nowrap="1"
                      data-img="/mass_mailing/static/src/img/theme_imgs/basic_thumb"
                      data-images-info='{"logo": {"format": "png"}}'>
-                    <t t-call="mass_mailing.theme_basic_template"/>
+                    <div>
+                        <p>
+                            <br/>
+                        </p>
+                    </div>
```

Comment:
I think we can keep the basic template view and put that simple DOM in it

Reply:
Now we are removing the default text then cann't we move that blank part in this instead of calling that template?

Reply:
I think I prefer consistency. As we are calling a template for the `default` theme and the enterprise themes, we should call one here too, even if it is nearly empty. This also allows users to modify that default template if they want to.